cmd/aws-controller/main_gen.go

```diff
@@ -184,10 +185,17 @@ func main() {
 
 		// create reconciler
 		reconciler := controller.Reconciler{
-			APIClient:        apiClient,
-			APIServer:        *apiServer,
-			ControllerID:     controllerID,
-			EncryptionKey:    encryptionKey,
+			APIClient:     apiClient,
+			APIServer:     *apiServer,
+			ControllerID:  controllerID,
+			EncryptionKey: encryptionKey,
+			EventsRecorder: &client_v1.EventRecorder{
+				APIClient:           apiClient,
+				APIServer:           *apiServer,
+				AttachedObjectType:  r.ObjectType,
+				ReportingController: "AwsController",
+				ReportingInstance:   os.Getenv("HOSTNAME"),
```

Comment:
Check if controllerID is tied to pod name's lifecycle, if not then let's add controllerID to `EventRecorder`

Reply:
Also add controllerID to `v0.Event`